app/src/main/java/com/stevesoltys/seedvault/restore/RestoreFilesFragment.kt

```diff
@@ -26,12 +28,16 @@ internal class RestoreFilesFragment : SnapshotFragment() {
 
         val topStub: ViewStub = v.findViewById(R.id.topStub)
         topStub.layoutResource = R.layout.header_snapshots
-        topStub.inflate()
+        val header = topStub.inflate()
+        val suw_layout: GlifLayout = header.findViewById(R.id.setup_wizard_layout)
```

Comment:
nitpick: variable names in Kotlin should use camelCase, no underscores.

edit: this also goes for all other places where this variable appears.